application/core/LimeMailer.php

```diff
@@ -767,17 +767,19 @@ public static function validateAddress($address, $patternselect = null)
         if (null === $patternselect) {
             $patternselect = static::$validator;
         }
-        if ($patternselect != 'php-idna') {
-            return parent::validateAddress($address, $patternselect);
-        }
-        require_once(APPPATH . 'third_party/idna-convert/idna_convert.class.php');
-        $oIdnConverter = new idna_convert();
-        $sEmailAddress = $oIdnConverter->encode($address);
-        $bResult = filter_var($sEmailAddress, FILTER_VALIDATE_EMAIL);
-        if ($bResult !== false) {
-            return true;
-        }
-        return false;
+        if ($patternselect == 'php-idna') {
+            /**
+             * PHPMailer has support for IDN, but it relies on `intl` and `mbstring`.
```

Comment:
We have `intl` and `mbstring` no ?

I create this when move from old system to LimeMailer  to use same fuinction, but maybeit's bet to use PJPMailer directly ?